Sources/BraveWallet/WalletStrings.swift

```diff
@@ -635,7 +635,7 @@ extension Strings {
       "wallet.passwordDoesNotMeetRequirementsError",
       tableName: "BraveWallet",
       bundle: .module,
-      value: "Passwords must be at least 7 characters, and contain at least one letter, one number, and one special character.",
```

Comment:
is `and contain at least one letter, one number, and one special character` not required anymore?

Reply:
> In particular, the following changes should be made:
>
> - minimum length must be 8 characters
> - different character types should not be required

https://github.com/brave/brave-browser/issues/22817